### PR TITLE
Revert 1570 file menu

### PR DESCRIPTION
**Pull request by arokem**
This supersedes #1577

### CODE REVIEW

**Comment by arokem** [comment]
Sorry folks: I don't understand the error that is currently happening on that one Travis bot (https://travis-ci.org/nipy/dipy/jobs/405653392). Can anyone provide some guidance for a fix?

File: dipy/viz/ui.py [hunk 1]
```diff
@@ -3024,12 +3023,12 @@ def _setup(self):
 
         # Add up and down buttons
         arrow_up = read_viz_icons(fname="arrow-up.png")
-        self.up_button = Button2D([("up", arrow_up)])
+        self.up_button = Button2D(("up", arrow_up))
```
Comment (skoudoro) [inline dipy/viz/ui.py]: I think the error comes from here. `Button2D([("up", arrow_up)])` is the right version. Can you confirm @dmreagan ?

File: dipy/viz/ui.py [hunk 2]
```diff
@@ -3024,12 +3023,12 @@ def _setup(self):
...
         pos = self.panel.size - self.up_button.size // 2 - margin
         self.panel.add_element(self.up_button, pos, anchor="center")
 
         arrow_down = read_viz_icons(fname="arrow-down.png")
-        self.down_button = Button2D([("down", arrow_down)])
+        self.down_button = Button2D(("down", arrow_down))
```
Comment (skoudoro) [inline dipy/viz/ui.py]: same here

**Comment by arokem** [comment]
Thanks. I'll change accordingly

File: dipy/viz/ui.py [hunk 3]
```diff
@@ -3029,7 +3028,7 @@ def _setup(self):
         self.panel.add_element(self.up_button, pos, anchor="center")
 
         arrow_down = read_viz_icons(fname="arrow-down.png")
-        self.down_button = Button2D([("down", arrow_down)])
+        self.down_button = Button2D([("down", arrow_down)]
```
Comment (skoudoro) [inline dipy/viz/ui.py]: small typo here, you forgot to close `)`. I am surprised that pep8speak did not complain.

**Comment by arokem** [comment]
Maybe it's given up on me.

**Comment by skoudoro** [comment]
I think it is ready to go @dmreagan! Can you have a last look and merge it?

**Comment by dmreagan** [comment]
Looks good. Merging. And now, @karandeepSJ, can you resubmit your FileMenu PR?

**Pull request merged**